service/history/task/transfer_active_task_executor.go

```diff
@@ -61,10 +61,9 @@ var (
 )
 
 var (
-	errUnknownTransferTask   = errors.New("unknown transfer task")
-	errWorkflowBusy          = errors.New("unable to get workflow execution lock within specified timeout")
-	errTargetDomainNotActive = errors.New("target domain not active")
```

Comment:
Note: Removing this category of error since it doesn't make sense without cross-cluster feature. 

I think the removal of this will make domain not active errors behave correctly during failover